.github/workflows/ci.yaml

```diff
@@ -10,12 +10,19 @@ jobs:
   unit-tests:
     name: Unit Tests
     runs-on: ubuntu-latest
+    services:
```

Comment:
@dpadhiar - please check the solution in Argo CD, and come up with proper approach for us.

For now we can use this.